pom.xml

```diff
@@ -26,26 +26,23 @@
     </licenses>
 
     <scm>
-        <connection>scm:git:https://github.com/${git.repo.slug}.git</connection>
-        <developerConnection>scm:git:git@github.com:${git.repo.slug}.git</developerConnection>
-        <url>https://github.com/${git.repo.slug}</url>
+        <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
+        <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
+        <url>https://github.com/${gitHubRepo}</url>
+        <tag>${scmTag}</tag>
     </scm>
 
     <properties>
-        <plugin.name>Jira Integration</plugin.name>
-        <plugin.description>Jenkins plugin for integrating with Jira Server, Data Center and Cloud.</plugin.description>
-        <documentation.url>https://github.com/${git.repo.slug}</documentation.url>
-        <git.repo.slug>jenkinsci/jira-integration-plugin</git.repo.slug>
+        <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
```

Comment:
`git-changelist-maven-extension` relies on this property being named `gitHubRepo`.